docs/_docs/usage.md

```diff
@@ -68,6 +68,10 @@ Make sure that your `$GOPATH/bin` is in your `$PATH`.
     }
    ```
 
+   See [a_bit_of_everything.proto](examples/proto/examplepb/a_bit_of_everything.proto)
```

Comment:
I don't know if there is some automated tooling syncing the README and the docs directory? I just put the content in both places.

Reply:
Yeah, we don't at the moment, the duplication is correct.